### ISSUE TITLE
New submissions for Tue, 10 Aug 21

### ISSUE THREAD

**Issue by alexandrethm** [alexandrethm/daily-arxiv#34]
## Keyword: data to text
There is no result 
## Keyword: data-to-text
There is no result 
## Keyword: data2text
There is no result 
## Keyword: graph2text
There is no result 
## Keyword: graph to text
There is no result 
## Keyword: graph-to-text
There is no result 
## Keyword: table to text
There is no result 
## Keyword: table-to-text
There is no result 
## Keyword: webnlg
There is no result 
## Keyword: genwiki
There is no result 
### #StayHome or #Marathon? Social Media Enhanced Pandemic Surveillance on  Spatial-temporal Dynamic Graphs
 - **Authors:** Yichao Zhou, Jyun-yu Jiang, Xiusi Chen, Wei Wang
 - **Subjects:** Social and Information Networks (cs.SI); Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2108.03670
 - **Pdf link:** https://arxiv.org/pdf/2108.03670
 - **Abstract**
 COVID-19 has caused lasting damage to almost every domain in public health, society, and economy. To monitor the pandemic trend, existing studies rely on the aggregation of traditional statistical models and epidemic spread theory. In other words, historical statistics of COVID-19, as well as the population mobility data, become the essential knowledge for monitoring the pandemic trend. However, these solutions can barely provide precise prediction and satisfactory explanations on the long-term disease surveillance while the ubiquitous social media resources can be the key enabler for solving this problem. For example, serious discussions may occur on social media before and after some breaking events take place. These events, such as marathon and parade, may impact the spread of the virus. To take advantage of the social media data, we propose a novel framework, Social Media enhAnced pandemic suRveillance Technique (SMART), which is composed of two modules: (i) information extraction module to construct heterogeneous knowledge graphs based on the extracted events and relationships among them; (ii) time series prediction module to provide both short-term and long-term forecasts of the confirmed cases and fatality at the state-level in the United States and to discover risk factors for COVID-19 interventions. Extensive experiments show that our method largely outperforms the state-of-the-art baselines by 7.3% and 7.4% in confirmed case/fatality prediction, respectively.
### Knowledge Graph Augmented Political Perspective Detection in News Media
 - **Authors:** Shangbin Feng, Zilong Chen, Qingyao Li, Minnan Luo
 - **Subjects:** Computation and Language (cs.CL); Artificial Intelligence (cs.AI); Computers and Society (cs.CY)
 - **Arxiv link:** https://arxiv.org/abs/2108.03861
 - **Pdf link:** https://arxiv.org/pdf/2108.03861
 - **Abstract**
 Identifying political perspective in news media has become an important task due to the rapid growth of political commentary and the increasingly polarized ideologies. Previous approaches only focus on leveraging the semantic information and leaves out the rich social and political context that helps individuals understand political stances. In this paper, we propose a perspective detection method that incorporates external knowledge of real-world politics. Specifically, we construct a contemporary political knowledge graph with 1,071 entities and 10,703 triples. We then build a heterogeneous information network for each news document that jointly models article semantics and external knowledge in knowledge graphs. Finally, we apply gated relational graph convolutional networks and conduct political perspective detection as graph-level classification. Extensive experiments show that our method achieves the best performance and outperforms state-of-the-art methods by 5.49\%. Numerous ablation studies further bear out the necessity of external knowledge and the effectiveness of our graph-based approach.
### Multilingual Compositional Wikidata Questions
 - **Authors:** Ruixiang Cui, Rahul Aralikatte, Heather Lent, Daniel Hershcovich
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2108.03509
 - **Pdf link:** https://arxiv.org/pdf/2108.03509
 - **Abstract**
 Semantic parsing allows humans to leverage vast knowledge resources through natural interaction. However, parsers are mostly designed for and evaluated on English resources, such as CFQ (Keysers et al., 2020), the current standard benchmark based on English data generated from grammar rules and oriented towards Freebase, an outdated knowledge base. We propose a method for creating a multilingual, parallel dataset of question-query pairs, grounded in Wikidata, and introduce such a dataset called Compositional Wikidata Questions (CWQ). We utilize this data to train and evaluate semantic parsers for Hebrew, Kannada, Chinese and English, to better understand the current strengths and weaknesses of multilingual semantic parsing. Experiments on zero-shot cross-lingual transfer demonstrate that models fail to generate valid queries even with pretrained multilingual encoders. Our methodology, dataset and results will facilitate future research on semantic parsing in more realistic and diverse settings than has been possible with existing resources.
### Facebook AI WMT21 News Translation Task Submission
 - **Authors:** Chau Tran, Shruti Bhosale, James Cross, Philipp Koehn, Sergey Edunov, Angela Fan
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2108.03265
 - **Pdf link:** https://arxiv.org/pdf/2108.03265
 - **Abstract**
 We describe Facebook's multilingual model submission to the WMT2021 shared task on news translation. We participate in 14 language directions: English to and from Czech, German, Hausa, Icelandic, Japanese, Russian, and Chinese. To develop systems covering all these directions, we focus on multilingual models. We utilize data from all available sources --- WMT, large-scale data mining, and in-domain backtranslation --- to create high quality bilingual and multilingual baselines. Subsequently, we investigate strategies for scaling multilingual model size, such that one system has sufficient capacity for high quality representations of all eight languages. Our final submission is an ensemble of dense and sparse Mixture-of-Expert multilingual translation models, followed by finetuning on in-domain news data and noisy channel reranking. Compared to previous year's winning submissions, our multilingual system improved the translation quality on all language directions, with an average improvement of 2.0 BLEU. In the WMT2021 task, our system ranks first in 10 directions based on automatic evaluation.
## Keyword: back-translation
There is no result 
## Keyword: back translation
There is no result 
## Keyword: cycle training
There is no result 
### Deep Learning Based Antenna-time Domain Channel Extrapolation for Hybrid  mmWave Massive MIMO
 - **Authors:** Shunbo Zhang, Shun Zhang, Jianpeng Ma, Tian Liu, Octavia A. Dobre
 - **Subjects:** Information Theory (cs.IT); Artificial Intelligence (cs.AI); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2108.03941
 - **Pdf link:** https://arxiv.org/pdf/2108.03941
 - **Abstract**
 In a time-varying massive multiple-input multipleoutput (MIMO) system, the acquisition of the downlink channel state information at the base station (BS) is a very challenging task due to the prohibitively high overheads associated with downlink training and uplink feedback. In this paper, we consider the hybrid precoding structure at BS and examine the antennatime domain channel extrapolation. We design a latent ordinary differential equation (ODE)-based network under the variational auto-encoder (VAE) framework to learn the mapping function from the partial uplink channels to the full downlink ones at the BS side. Specifically, the gated recurrent unit is adopted for the encoder and the fully-connected neural network is used for the decoder. The end-to-end learning is utilized to optimize the network parameters. Simulation results show that the designed network can efficiently infer the full downlink channels from the partial uplink ones, which can significantly reduce the channel training overhead.
